src/Tests/dotnet-add-reference.Tests/GivenDotnetAddReference.cs

```diff
@@ -161,11 +163,26 @@ public void WhenBrokenProjectIsPassedItPrintsErrorAndUsage()
             string projName = "Broken/Broken.csproj";
             var setup = Setup();
 
+            string brokenFolder = Path.Combine(setup.TestRoot, "Broken");
+            Directory.CreateDirectory(brokenFolder);
+            string brokenProjectPath = Path.Combine(brokenFolder, "Broken.csproj");
+            File.WriteAllText(brokenProjectPath, @"<Project Sdk=""Microsoft.NET.Sdk"" ToolsVersion=""15.0"" xmlns=""http://schemas.microsoft.com/developer/msbuild/2003"">
+    < PropertyGroup >
```

Comment:
VS added extra spaces here when you pasted (as well as in the other two locations).